pkgs/development/libraries/gsignond/plugins/oauth.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchFromGitLab, pkgconfig, meson, ninja, glib, gsignond, check
+, json-glib, libsoup, gnutls, gtk-doc, docbook_xml_dtd_43, docbook_xml_dtd_45
+, docbook_xsl, glibcLocales, gobjectIntrospection }:
+
+stdenv.mkDerivation rec {
+  name = "gsignond-plugin-oauth-${version}";
+  version = "2018-05-13";
+
+  src = fetchFromGitLab {
+    owner = "accounts-sso";
+    repo = "gsignond-plugin-oa";
+    rev = "43fee492ef84533ea2282a161b2695e0a95eb186";
+    sha256 = "00khg6fh2gmi8iz068dmdzj7mfg4jzay628r1z46iqbzhkq8ny7w";
+  };
+
+  nativeBuildInputs = [
+    check
```

Comment:
This might go to `checkInputs` which were recently added to `mkDerivation`: https://github.com/NixOS/nixpkgs/commit/87651b32fe44b9cdeb5dc55d6197d5c5158ce42a

Reply:
I get this when I do that `meson.build:9:0: ERROR:  Native dependency 'check' not found`

Reply:
Right, they are only added with `doCheck = true`

Reply:
I've tried this and ninja just stalls.